Extension/src/common.ts

```diff
@@ -333,6 +333,15 @@ export function isArrayOfString(input: any): input is string[] {
     return isArray(input) && input.every(isString);
 }
 
+// This helper function validates whether the given object is a valid mapping of key and value type.
```

Comment:
I believe the recommended commenting style is to not use "This helper function" or "This <whatever>" and instead just use "Validates whether..." directly. Asked Copilot and it agreed with me and gave this example:

e.g. 
// Computes the sum of 2 integers.

versus

// This function computes the sum of 2 integers.